src/repository/storage/postgres/orm/sequelize/user.ts

```diff
@@ -0,0 +1,86 @@
+import { Model, DataTypes, Sequelize, InferAttributes, InferCreationAttributes, CreationOptional } from 'sequelize';
+import Orm from '@repository/storage/postgres/orm/sequelize/index.js';
+
+/* eslint-disable @typescript-eslint/naming-convention */
+
+/**
+ * Class representing a user model in database
+ */
+export class UserModel extends Model<InferAttributes<UserModel>, InferCreationAttributes<UserModel>> {
+  /**
+   * User id
+   */
+  public declare id: CreationOptional<number>;
+
+  /**
+   * User email address
+   */
+  public declare email: string;
+
+  /**
+   * User name
+   */
+  public declare name: string;
+
+  /**
+   * User creation date
+   */
+  public declare created_at: Date;
```

Comment:
why underscore instead if camelCase?

Reply:
Because as i know, it is not comfortable to use camelCase fields in tables. If you use it, you need to wrap names in quotes, like `select users."createdAt" from users` instead of `select users.created_at from users`, when you make queries